code/datums/vr/level.dm

```diff
@@ -1,16 +1,117 @@
+//death types VR_DROP_NONE, VR_DROP_ALL, VR_DROP_BLACKLIST, VR_DROP_WHITELIST
+
+
 /datum/map_template/vr/level
 	prefix = "_maps/map_files/vr/"
+	suffix = null
+	name = "Error Error"
+	description = "You should not see this."
+	death_type = VR_DROP_NONE
+	drop_whitelist = null
+	drop_blacklist = null
```

Comment:
Won't these vars (`drop_whitelist`, `death_type`, `drop_blacklist`) just be inherited from `/datum/map_template/vr` where you've already defined them as `null`? No need to redefine them here.